pkg/storaged/stratis-details.jsx

```diff
@@ -368,25 +372,45 @@ export const StratisPoolDetails = ({ client, pool }) => {
                 CheckBoxes("mount_options", _("Mount options"),
                            {
                                value: {
-                                   auto: true,
                                    ro: false,
-                                   never_auto: false,
                                    extra: false
                                },
                                fields: [
-                                   { title: _("Mount now"), tag: "auto" },
                                    { title: _("Mount read only"), tag: "ro" },
-                                   {
-                                       title: _("Never mount at boot"),
-                                       tag: "never_auto",
-                                       tooltip: never_auto_explanation,
-                                   },
                                    { title: _("Custom mount options"), tag: "extra", type: "checkboxWithInput" },
                                ]
                            }),
+                SelectOne("at_boot", _("At boot"),
+                          {
+                              value: "nofail",
+                              explanation: mount_explanation.nofail,
+                              choices: [
+                                  {
+                                      value: "local",
+                                      title: _("Mount before services start"),
+                                  },
+                                  {
+                                      value: "nofail",
+                                      title: _("Mount without waiting, ignore failure"),
+                                  },
+                                  {
+                                      value: "netdev",
+                                      title: _("Mount after network becomes available, ignore failure"),
+                                  },
+                                  {
+                                      value: "never",
+                                      title: _("Do not mount"),
+                                  },
+                              ]
+                          }),
             ],
+            update: function (dlg, vals, trigger) {
+                if (trigger == "at_boot")
+                    dlg.set_options("at_boot", { explanation: mount_explanation[vals.at_boot] });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221115-130151-a219e579-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)